### PR TITLE
corrected namespace

### PR DESCRIPTION
**Pull request by AnTAVR**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: tests/codeception/backend/acceptance/LoginCept.php [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <?php
 
+namespace tests\codeception\backend\acceptance;
```
Comment (samdark) [inline tests/codeception/backend/acceptance/LoginCept.php]: It's not required for Codeception to work.

Comment (samdark) [inline tests/codeception/backend/acceptance/LoginCept.php]: Have you actually checked if it works w/ this change?

Comment (AnTAVR) [inline tests/codeception/backend/acceptance/LoginCept.php]: Все работает, тесты проходят.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**